src/output.cpp

```diff
@@ -638,18 +638,23 @@ write_tallies()
   for (auto i_tally = 0; i_tally < model::tallies.size(); ++i_tally) {
     const auto& tally {*model::tallies[i_tally]};
 
+    // Write header block.
+    std::string tally_header("TALLY " + std::to_string(tally.id_));
+    if (!tally.name_.empty()) tally_header += ": " + tally.name_;
+    tallies_out << header(tally_header) << "\n\n";
+
+    if (!tally.writeable_) {
```

Comment:
Maybe we should just skip the tally altogether here?

Reply:
I thought about that. The reason I marked it as internal is the tally numbers still increase by one. So someone would see "Tally 1...Tally 2...Tally 4..." and could be confusing.

Alternatively, we could switch the iteration structure to iterate over tallies and then increment the counter when a tally is actually written.